invenio/modules/deposit/validation_utils.py

```diff
@@ -165,7 +165,7 @@ def __init__(self, message=None):
         self.regexp = re.compile(self.pattern, re.I)
         self.message = message if message else (
             "The provided DOI is invalid - it should look similar to "
-            "'10.1234/foo.bar'.")
```

Comment:
I don't think linking to a random paper is better than the current invalid DOI. Especially since the paper is an astrophysics paper and we then forces everyone who uses the validator (e.g. in social sciences) to display an astrophysics paper. I suggest one of these alternatives instead:

* You can provide a custom message on the INSPIRE site ``DOISyntaxValidator("my message")``.
* Use a test prefix DOI (starting with ``10.5072``) which has already been used as an example (e.g. ``10.5072/example-full``). Here's a couple of more options: http://schema.datacite.org/meta/kernel-3.1/index.html
* Use the DOI of e.g. DataCite Metadata Schema documentation (``10.5438/0010``).